acs-integration-tests/pom.xml

```diff
@@ -658,8 +658,53 @@
                             <groupId>commons-io</groupId>
                             <artifactId>commons-io</artifactId>
                         </exclusion>
+                        <exclusion>
+                            <groupId>io.netty</groupId>
+                            <artifactId>netty-handler</artifactId>
+                        </exclusion>
+                        <exclusion>
+                            <groupId>io.netty</groupId>
+                            <artifactId>netty-codec-http</artifactId>
+                        </exclusion>
                     </exclusions>
                 </dependency>
+                <dependency>
+                    <groupId>com.ge.predix.eventhub</groupId>
+                    <artifactId>predix-event-hub-sdk</artifactId>
```

Comment:
They're the same Audit-related dependencies and exclusions as the `predix-local` profile above it and are necessary so that Maven Enforcer doesn't crib when running integration tests against the `predix-local-titan` profile